internal/oci/runtime_oci.go

```diff
@@ -851,11 +857,28 @@ func (r *runtimeOCI) StopLoopForContainer(c *Container) {
 		}
 	}()
 
+	// Keeps a track of which ticker interval is currently used.
+	tickerIndex := 0
+
+	// Get progressively increasing ticker duration. When the index reaches
+	// the last value, it will wrap around, allowing for a shorter interval
+	// to be used again and, as such, for burst periods.
+	getTickerInterval := func() time.Duration {
+		tickerIndex %= len(stopTickerIntervals)
+		t := time.Duration(stopTickerIntervals[tickerIndex]*1000) * time.Millisecond
+		tickerIndex++
+		return t
+	}
+
+	ticker := time.NewTicker(getTickerInterval())
```

Comment:
I think you can use the ExponentialBackoff in k8s.io/apimachinery/pkg/util/wait/ which I believe will simplify this  a bit

Reply:
@haircommander, can do. 👍 

I looked at this first, as we use it throughout the code base, but then decided that having a ability to burst and then also to roll-over to eventually try shorter intervals again and burst, was an interesting idea. Simple to add too.

That said, I will convert it. So we will have progression with an increasing internal between retries up to the limit of 5 minutes, from where it will always use 5 minutes as an interval going forward.

Reply:
Done.